src/cookbooks/publishing_datasets.md

```diff
@@ -18,30 +18,31 @@ owners:
   - example@mozilla.com
 labels:
   application: firefox
-  incremental: true     # incremental queries add data to existing tables
-  schedule: daily       # scheduled in Airflow to run daily
+  incremental: true # incremental queries add data to existing tables
```

Comment:
Ditto - the comments were aligned vertically before, making it easier to read them.

Reply:
Looks like someone else had the same opinion: prettier/prettier#8690

I don't think this is really worth worrying about. It's still quite readable IMO.